content/code_samples/escrow/websocket/account_objects-request.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": 5,
   "command": "account_objects",
-  "account": "rEhw9vD98ZrkY4tZPvkZst5H18RysqFdaB",
+  "account": "rfztBskAVszuS3s5Kq7zDS74QtHrw893fm",
   "ledger_index": "validated",
   "type": "escrow"
 }
```

Comment:
This file replaces account_objects_request_sender.json and serves as the request code sample for both sender and receiver.